swarm/api/http/server.go

```diff
@@ -43,6 +43,12 @@ import (
 	"github.com/rs/cors"
 )
 
+type resourceResponse struct {
```

Comment:
why do you need explicit json tags?

Reply:
In case we want all this data returned. Maybe we can make it contingent on the `Accept` header?